src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,32 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100, hidden = false) {
+    this.name = name;
+    this.health = health;
+    this.hidden = hidden;
```

Comment:
The constructor of the `Animal` class should not have a `hidden` parameter since not all animals will use this property. It's better to define `hidden` directly in the `Herbivore` subclass where it's relevant.